### DISCUSSION TITLE
Details regarding parameter use_other_treat_as_covariate in DoubleMLData?

### DISCUSSION THREAD

**Discussion by yangfanliang** [DoubleML/doubleml-for-py#146] [Q&A] [answered]
Right now I am applying double machine learning on a setting where there are multiple treatments, and I notice parameter _use_other_treat_as_covariate_ in DoubleMLData. I tried to find detailed documentation for this argument in both documentation and your papers, but the best I can get is the following: "use_other_treat_as_covariate (bool) – Indicates whether in the multiple-treatment case the other treatment variables should be added as covariates. Default is True." If possible, could you please share some details regarding this parameter? Such as when should we set this as true? 

Thanks in advance for any suggestions!

**Comment by MalteKurz** [comment, marked as answer]
Thanks for your interest in `DoublemL`. Some documentation on the multiple treatment case can be found here https://docs.doubleml.org/stable/guide/sim_inf.html. The implementation is closely related to [Belloni et al. (2018)](https://doi.org/10.1214/17-AOS1671). For each treatment effect that should be estimated there is one identifying orthogonal moment condition. If you then estimate nuisance functions, they are allowed to depend on X and all other treatment variables (see the second and third equation in the above linked documentation or [Belloni et al. (2018)](https://doi.org/10.1214/17-AOS1671)). If `use_other_treat_as_covariate` is `True` the nuisance functions are estimated with all the `X` variables and all other treatment variables `D` as covariates. If `use_other_treat_as_covariate` is `False`, only the `X` variables are used. Usually, I would recommend to set `use_other_treat_as_covariate` to `True`, because even if the other treatment variables are not having any effect, ML models for the nuisance function approximation usually can handle these additional irrelevant variables.

**Reply by yangfanliang** [reply]
That is very helpful, thanks a lot for your prompt reply! 

One possible suggestion: It might be easier for the users to find relevant information if you could add [https://docs.doubleml.org/stable/guide/sim_inf.html](https://docs.doubleml.org/stable/guide/sim_inf.html) into the API documentation of [DoubleMLData](https://docs.doubleml.org/stable/api/generated/doubleml.DoubleMLData.html).
